### ISSUE TITLE
[Test Failure][Accessibility] Package source drop down and settings button truncated when selecting ‘Microsoft Visual Studio Offline Packages’ in PM UI

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#4502]
**NuGet Version:** Dev 2265 VS15 Localized NuGet Client Vsix
**VS Version:** D15Rel 26203.00 (select all languages and workloads)
**OS Version:** Windows 10 RS1_Release 14393.821.170127-1801

**Repro Steps:**
1.  Select **CSY** through ‘Tools -> Options -> International Settings’.
2.  Create a new C# Console Application via VS.
3.  Right-click on the project and click ‘Manage NuGet Packages…’.
4.  Select **‘Microsoft Visual Studio Offline Packages’** on the right-top of PM UI.

**Expected Result:**
Package source drop down and settings button won’t truncated by the Solution Explorer Window on the right.

**Actual Result:**
Package source drop down and settings button truncated by the Solution Explorer Window on the right as below screenshot.
![packagesourcetruncated](https://cloud.githubusercontent.com/assets/20632625/22618305/0e87d458-eb14-11e6-89c5-db488c8cf435.png)

**Note:**
1. It repro on CSY,PTB,DEU,RUS,ESN,FRA,ITA,PLK.
2. It don’t repro on TRK,KOR,CHS,CHT,JPN.

**Comment by mishra14** [comment]
This too seems like an Accessibility bug.

**Comment by v-luzh** [comment]
This also repro on OS: Win10 RTM RUS+VS 2015 U3 RUS with default window layout.
![4502reproondev14](https://cloud.githubusercontent.com/assets/20632625/23011147/3af14778-f45a-11e6-8c06-86192c70ebb2.PNG)

**Comment by v-luzh** [comment]
Fixed on D15.8 27719.00 + NuGet Client Dev 4.8.0.5215.
